README.md

```diff
@@ -25,6 +25,8 @@ All active versions of python3 are supported.
 
 The `xr-compose` script also requires `docker-compose` (v1) to be on `PATH` and for the `PyYAML` python package be installed (e.g. in an active virtual environment).
 
+Podman/docker is required to build a `host-check` container image from the Dockerfile.
```

Comment:
I'd err on the side of omitting this - if a user is building an image for host-check, then they probably already know what they are doing; it's hard to enumerate all the options (e.g., buildah, classic docker, docker buildx, kaniko, etc...).